phpBB/styles/prosilver/template/ajax.js

```diff
@@ -138,9 +138,9 @@ phpbb.markNotifications = function($popup, unreadCount) {
 	}
 
 	// Update page title
-	$('title').text(
-		(unreadCount ? '(' + unreadCount + ')' : '') + $('title').text().replace(/(\(([0-9])\))/, '')
-	);
+	var $title = $('title');
+	var originalTitle = $title.text().replace(/(\((\d+)\))/, '');
+	$title.text((unreadCount ? '(' + unreadCount + ')' : '') + originalTitle);
```

Comment:
I think the code above was buggy; looks like when the number of notifications goes above ten, it doesn't remove the old notification count